diff --git a/node_modules/react-native-reanimated/plugin-standalone.js b/node_modules/react-native-reanimated/plugin-standalone.js
index 2fb0c21..050cc8b 100644
--- a/node_modules/react-native-reanimated/plugin-standalone.js
+++ b/node_modules/react-native-reanimated/plugin-standalone.js
@@ -1,8 +1,12 @@
 'use strict';
-const generate = require('@babel/generator').default;
+
+// const generate = require('@babel/generator').default;
 const hash = require('string-hash-64');
-const traverse = require('@babel/traverse').default;
-const { transformSync } = require('@babel/core');
+// const traverse = require('@babel/traverse').default;
+// const { transformSync } = require('@babel/core');
+
+const { generate, traverse, transformSync } = require('snack-babel-standalone');
+
 /**
  * holds a map of function names as keys and array of argument indexes as values which should be automatically workletized(they have to be functions)(starting from 0)
  */
